### ISSUE TITLE
Validation of image/container  before running test.

### ISSUE THREAD

**Issue by kunalkushwaha** [estesp/bucketbench#1]
A single threaded benchmark also output large output logs, in case containers are not created due to any error.

A validation for single unit (``run-stop-delete``) required before running  ``bench.Run()``.

e.g output
```
WARN[0079] BasicBench: run error on container "bb-ctr-0-0": exit status 1
  Output:  
WARN[0079] BasicBench: stop error on container "bb-ctr-0-0": exit status 1
  Output: container "bb-ctr-0-0" does not exist
 
WARN[0079] BasicBench: remove error on container "bb-ctr-0-0": exit status 1
  Output: container bb-ctr-0-0 does not exist
one or more of the container deletions failed
 
WARN[0079] BasicBench: run error on container "bb-ctr-0-1": exit status 1
.
.
.
.
WARN[0082] BasicBench: remove error on container "bb-ctr-0-48": exit status 1
  Output: container bb-ctr-0-48 does not exist
one or more of the container deletions failed
 
WARN[0082] BasicBench: run error on container "bb-ctr-0-49": exit status 1
  Output:  
WARN[0082] BasicBench: stop error on container "bb-ctr-0-49": exit status 1
  Output: container "bb-ctr-0-49" does not exist
 
WARN[0082] BasicBench: remove error on container "bb-ctr-0-49": exit status 1
  Output: container bb-ctr-0-49 does not exist
one or more of the container deletions failed
 
             Iter/Thd     1 thrd  2 thrds  3 thrds  4 thrds  5 thrds  6 thrds  7 thrds  8 thrds  9 thrds 10 thrds
Limit            1000     192.36   430.19   606.81   780.29   788.20   753.38   741.87   715.35   767.91   774.87
RuncBasic          50      15.16

```

**Comment by estesp** [comment]
This definitely makes sense.. and I see your attempt in #2. I wonder if it makes more sense to add this test to the existing `Init()`? I will comment more in the PR.

**Comment by estesp** [comment]
Fixed by #2
